onnxruntime/python/tools/transformers/models/stable_diffusion/benchmark.py

```diff
@@ -43,7 +44,9 @@ def example_prompts():
         "delicate elvish moonstone necklace on a velvet background, symmetrical intricate motifs, leaves, flowers, 8k",
     ]
 
-    return prompts
+    negative_prompt = "bad composition, ugly, abnormal, malformed"
+
+    return prompts, negative_prompt
 
 
 class CudaMemoryMonitor:
```

Comment:
Can the memory monitors be imported from [here](https://github.com/microsoft/onnxruntime/blob/ef94b1b7fadaa8a6eef64e737ff04c32203cb698/onnxruntime/python/tools/transformers/benchmark_helper.py#L465) and modified for stable diffusion as needed?